cohort_4/playbook/activities.md

```diff
@@ -58,6 +61,31 @@ first floor with the elevators.
 
 TODO
 
+## Demos
+
+### Presenting your demo
```

Comment:
Perhaps there could be a more descriptive title? Remember, this is facilitator facing.

One idea may be to get rid of this heading entirely and just have all this info directly under the `## Demos` heading.